styles/sass/base/_page.scss

```diff
@@ -21,4 +21,8 @@ body {
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+
+  @include breakpoint(mobile) {
+    width: calc(100vw - 32px);
+  }
```

Comment:
it' might look cleaner if  you use CSS variables. 

```css
body {
 --margin-x: 32;
  width: calc(100vw - 2px * var(--margin-x));

  @include breakpoint(mobile) {
    --margin-x: 16;
  }
}
```
just an idea

Reply:
I have to think